test/createAndSendTx.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { typesBundle } from "moonbeam-types-bundle";
+import { createAndSendTx } from "../src/methods/createAndSendTx";
+import { ALITH, BALTATHAR, testnetWs } from "../src/methods/utils";
+import { testSignCLI } from "./sign.spec";
+var assert = require("assert");
+
+const testAmount = "1000000000000";
+
+async function getBalance(address: string, api: ApiPromise) {
+  const account = await api.query.system.account(address);
+  return account.data.free.toString();
+}
+
+describe("Create and Send Tx Integration Test", function () {
+  it("should increment Baltathar's account balance", async function () {
+    this.timeout(40000);
+    let api = await ApiPromise.create({
```

Comment:
I don't think we should test against a real network. Specially if we are sending transaction.
Moreover this test relies on a real account, the probability that the account will have funds to pass the CI is going to be very low.

Reply:
(To clarify in case it wasn't obvious, this repo is (maybe not yet?) public and the first person that reads the test will try to take the token out of the account. And because they are public accounts, people might look for those in other repos when we publish the key.

Reply:
How do you recommend I fetch the moonbeam binary for dev testing?